index.html

```diff
@@ -408,6 +408,7 @@ <h3>
             <li>Use case and requirement documents;</li>
             <li>Test suite and implementation report for the specification;</li>
             <li>Primer or Best Practice documents to support web developers when designing applications.</li>
+            <li>In particular, an updated version of the <a href="https://www.w3.org/TR/rdf11-primer/">RDF 1.1 Primer</a>;
```

Comment:
Not blocking: remove "In particular,"

Reply:
I put "In particular" because this item is redundant with the previous one, which already encompasses primer. Or should I then merge the two items?

Reply:
As a list, it reads oddly to me.
```
     <li>Primer or Best Practice documents to support web developers when designing applications, 
             including updating the <a href="https://www.w3.org/TR/rdf11-primer/">RDF 1.1 Primer</a></li>
```